src/olympia/addons/migrations/0004_auto_20191126_1712.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.6 on 2019-11-26 17:12
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('addons', '0003_addonreviewerflags_auto_approval_disabled_until'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='addon',
+            name='type',
+            field=models.PositiveIntegerField(choices=[(1, 'Extension'), (2, 'Deprecated Complete Theme'), (3, 'Dictionary'), (4, 'Search Engine'), (5, 'Language Pack (Application)'), (6, 'Language Pack (Add-on)'), (7, 'Plugin'), (9, 'Deprecated LWT'), (10, 'Theme (Static)')], db_column='addontype_id', default=1),
```

Comment:
Pretty sure I didn't make this change in the patch, but it seems pretty harmless.

Reply:
I think I agree, same with `category.type` below which is curious. The values are identical to `ADDON_TYPE` so that should be fine I think. Do you have maybe some local changes that you didn't commit?

Reply:
I don't.  Don't you get the same thing if you do `./manage.py makemigrations addons` ?

Reply:
I get the same migrations from a checkout from mozilla/addons-server:master.  Possibly from when I changed the status labels.